lib/bemhtml/index.js

```diff
@@ -69,13 +69,11 @@ BEMHTML.prototype.render = function render(context,
     if (js === true)
       js = {};
 
-    if (js) {
+    if (js && js !== ctx.js) {
       if (ctxJS !== true)
         js = utils.extend(ctxJS, js);
     }  else if (ctxJS === true) {
       js = {};
-    } else {
-      js = ctxJS;
```

Comment:
Yep. Because of [this.js.exec(context)](https://github.com/bem/bem-xjst/blob/master/lib/bemhtml/entity.js#L64) now returns `this.ctx.js` by default.